tests/components/netatmo/test_init.py

```diff
@@ -121,7 +121,7 @@ async def fake_post(*args, **kwargs):
 
         await hass.async_block_till_done()
 
-        assert fake_post_hits == 10
+        assert fake_post_hits >= 8
```

Comment:
I'm not sure what this is trying to test. Its relying on event loop timing though so I'm not sure how useful the test here is.